denis.roman/1.rb

```diff
@@ -0,0 +1,14 @@
+require 'faraday'
+
+class Image
+
+  def self.download(url)
+    response = Faraday.get url
+    extension = response.headers["content-type"].split('/').last
+    File.open("download.#{extension}", 'w') { |fp| fp.write(response.body) }
+    raise StandardError, 'file is not an image' if extension != 'png'
```

Comment:
Better use here `TypeError` instead `StandardError` and add here others image formats
In addition to the format error, you had to add `ArgumentError` with response status checking `response.status`